monitoring/metrics/index.md

```diff
@@ -38,6 +38,23 @@ To enable collecting metrics in an environment:
 NOTE: [Monitoring NServiceBus solutions: Getting started](/tutorials/monitoring-setup/) is an in-depth, step-by-step tutorial about installing and configuring everything to get the most out of performance monitoring.
 
 
+## Performance impact on system resources
+
+A ServiceControl monitoring instance is very lightweight compares a regular ServiceControl instance. We would still advice not to host a monitoring instance on the same machine that is hosting any production endpoint instances.
```

Comment:
> A ServiceControl monitoring instance is very lightweight compares a regular ServiceControl instance

Should be probably

> A ServiceControl monitoring instance is much lighter than a regular ServiceControl instance.

Reply:
For my curiosity. Why wouldn't we recommend running it on the same box as endpoints?

Reply:
@danielmarbach @Scooletz That is the advice in general to not host any platform tool components on the same box so that these cannot affect any production resources.